src/routes/unsupported.nim

```diff
@@ -11,10 +11,12 @@ proc createUnsupportedRouter*(cfg: Config) =
       resp renderMain(renderFeature(), request, cfg, themePrefs())
 
     get "/about/feature": feature()
-    get "/intent/?@i?": feature()
+    get "/intent/?@i?": 
```

Comment:
Now that this has a body with more than one line it should go below the `@name/lists` route with empty lines around it